### PR TITLE
Optimization:Remove 2 Feed Experiments

### PR DESCRIPTION
**Pull request by mstruve**
## What type of PR is this? (check all applicable)
- [x] Optimization

## Description
We have been running these feed experiments for a while now and recently they came up in a PR. I took the initiative to start closing the loop on them. The reason I want to close the loop on this is that multiple code paths add complexity to our codebase and additional time to our specs when we test them. If we can narrow down to the ideal paths(doesn't even have to be 1!) then we can begin to optimize those paths for performance and other Forems. 

I recently looked [at the results of our A/B tests](https://dev.to/abtests) and found that because we have so many experiments running it's pretty hard to tell if any are the "true winner". In this case, I took a couple of experiments that seemed to have lower or very middle of the pack results and removed them. After allowing these tests to run with the new configurations we can check the results again in a week or two and continue to narrow down the scope of the experiment. 

## Related Tickets & Documents
https://github.com/orgs/forem/projects/6#card-47841833

## Added tests?
- [x] no, because they aren't needed

alt_text

### CODE REVIEW

**Review by maestromac — APPROVED** [review]

File: app/services/articles/feeds/large_forem_experimental.rb [hunk 1]
```diff
@@ -118,30 +102,26 @@ def more_experience_level_weight_experiment
...
           more_experience_level_weight_randomized_at_end_experiment
-        when 9
+        when 7
           more_comments_randomized_at_end_experiment
-        when 10
+        when 8
           more_comments_medium_weight_randomized_at_end_experiment
-        when 11
+        when 9
           more_comments_minimal_weight_randomized_at_end_experiment
```
Comment (vaidehijoshi) [inline app/services/articles/feeds/large_forem_experimental.rb:125]: Hm, just seeing this again makes me 😩 I wonder how we can improve this.

**Review by vaidehijoshi — APPROVED** [review]
Thanks for this!!

File: app/services/articles/feeds/large_forem_experimental.rb (hunk 1 above)
Comment (mstruve) [reply app/services/articles/feeds/large_forem_experimental.rb:125]: You and me both which is why I decided to take ownership of it. We will get there!

Comment (citizen428) [reply app/services/articles/feeds/large_forem_experimental.rb:125]: How about this?

Wrap `default_home_feed` in an `_experiment` method:

```ruby
def default_home_feed_experiment
  default_home_feed(user_signed_in: true)
end
```

Add this method to the `VARIANTS` constant and this becomes

```ruby
variant_method = VARIANTS.values.sample
public_send(variant_method)
```

If we don't want to add the wrapper, we can at least reduce it this to:

```ruby
if rand(10).zero?
  default_home_feed(user_signed_in: true)
else
  variant_method = VARIANTS.values.sample
  public_send(variant_method)
end
```

I prefer the first option with an explicit `default_home_feed_experiment`.

Comment (mstruve) [reply app/services/articles/feeds/large_forem_experimental.rb:125]: @citizen428 my plan is to leave everything as is until we have "final paths" and then figure out the best way to handle those.

**Review by rhymes — APPROVED** [review]

**Pull request merged**